docs/using-workflows/adding-connections.md

```diff
@@ -22,3 +24,24 @@ To set the required values for the connection, on the workflow's page, expand th
 ## Managing connections 
 Connections can be managed from the Connections menu option on the left nav. You can add new connections, override the values for existing connections, or delete existing connections. 
 
+## About Connections
+Connections are encrypted when entered and remain encrypted until they are used by a workflow.
```

Comment:
watch the passive voice here, how about:
Relay encrypts connections when you create them and keeps them secure until a workflow needs them.